worker-markup-core/src/main/java/com/github/cafdataprocessing/worker/markup/core/EmailSquash.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cafdataprocessing.worker.markup.core;
+
+import org.jdom2.Element;
+
+import java.util.List;
+
+/**
+ * Created by Morvin Shah on 6/14/2018.
+ */
+public final class EmailSquash {
+
```

Comment:
maybe try and think of a better name that conveys what the class is being used for